tests/fixtures/kobo-client-get-404.yml

```diff
@@ -18,7 +18,7 @@ http_interactions:
     headers:
       status: HTTP/1.1 404 Not Found
       server: nginx
-      date: Mon, 10 May 2021 13:29:49 GMT
+      date: Sun, 16 May 2021 19:03:34 GMT
```

Comment:
I am not sure how the testing works here, so apologies if what follows do not make sense. 
Should we commit this file? There are dates that I guess they will change at any run.
Should this file be added to gitignore or is it a template for the tests?
This would apply also to  tests/fixtures/kobo-client-403.yml

Reply:
nope they only have to be re-run when the request you make in the test changes or if you choose so. sorry that i still haven't written the vignette on this, will do this on the weekend. 
so the date only changed because i had to adapt the test a final time and then had to re-run it. i also could've not committed this change but somehow was too lazy to revert the change and just committed it ;)